src/shared/components/mutationMapper/DriverAnnotationProteinImpactTypeBadgeSelector.tsx

```diff
@@ -241,33 +260,28 @@ export default class DriverAnnotationProteinImpactTypeBadgeSelector extends Prot
         })).filter(
             type =>
                 !(
-                    this.props.excludedProteinTypes?.includes(type.value) ||
-                    (type.value === ProteinImpactType.OTHER &&
-                        type.badgeContent === 0)
+                    excludedProteinTypes.includes(type.value) ||
```

Comment:
lets do the filter operation before the map operation above since it only requires value.  the code will be clearer.